src/patternfly/components/Switch/switch.scss

```diff
@@ -149,7 +144,6 @@
   position: absolute;
   top: var(--pf-c-switch__toggle-icon--Top);
```

Comment:
Looking at the react switch, the icon is misaligned - https://www.patternfly.org/v4/documentation/react/components/switch/

One way to fix it would be to replace this block with below and remove `--pf-c-switch__toggle-icon--Top` and `--pf-c-switch__toggle-icon--Transform`

```
.pf-c-switch__toggle-icon {
  position: absolute;
  top: 0;
  bottom: 0;
  left: var(--pf-c-switch__toggle-icon--Left);
  display: flex;
  align-items: center;
  padding-left: var(--pf-c-switch__toggle-icon--PaddingLeft);
  font-size: var(--pf-c-switch__toggle-icon--FontSize);
  color: var(--pf-c-switch__toggle-icon--Color);
}
```